cobalt/web/agent.cc

```diff
@@ -560,6 +575,22 @@ void Agent::Run(const Options& options, InitializeCallback initialize_callback,
   if (!thread_.StartWithOptions(thread_options)) return;
   DCHECK(message_loop());
 
+  watchdog::Watchdog* watchdog = watchdog::Watchdog::GetInstance();
+
+  // Registers service worker thread as a watchdog client.
+  if (watchdog) {
+    watchdog_name_ =
+        thread_.thread_name() + std::to_string(thread_.GetThreadId());
+    watchdog_registered_ = true;
+    watchdog->Register(watchdog_name_, watchdog_name_,
+                       base::kApplicationStateStarted, kWatchdogTimeInterval,
+                       kWatchdogTimeWait, watchdog::PING);
+    message_loop()->task_runner()->PostDelayedTask(
+        FROM_HERE,
+        base::Bind(&Agent::PingWatchdog, base::Unretained(this), watchdog),
+        base::TimeDelta::FromMilliseconds(5000));
```

Comment:
This should probably be a 'const int' at the top of this file, here, and in PingWatchdog below, and also in the service worker version of watchdog integration.